Dependencies/xr/Source/OpenXR/XrRegistry.h

```diff
@@ -1,25 +1,31 @@
 #pragma once
 
 #include "XrSupportedExtensions.h"
+#include "Include/IXrContext.h"
 
 namespace xr
 {
     struct SceneUnderstanding;
 
-    struct XrSessionContext
+    struct XrSessionContext : public IXrContext
     {
         XrSessionContext();
         ~XrSessionContext();
-        bool IsInitialized() const;
-        XrInstance Instance() const;
-        XrSystemId SystemId() const;
-        XrTime DisplayTime() const;
-        const std::unique_ptr<XrSupportedExtensions>& Extensions() const;
-        const XrSession Session() const;
-        const XrSessionState State() const;
-        const XrSpace Space() const;
+
+        virtual bool OPENXR_CONTEXT_INTERFACE_API IsInitialized() const override;

```

Comment:
>virtual [](start = 8, length = 7)

nit: `override` implies `virtual` so this is extraneous. Not sure what the style preferences are for this repo though.